Source/x11/XGServerWindow.m

```diff
@@ -4017,55 +4075,9 @@ - (void) _DPSsetcursor: (Cursor)c : (BOOL)set
     }
 }
 
-#define ALPHA_THRESHOLD 158
+#if !HAVE_XCURSOR
 
-Pixmap
-xgps_cursor_mask(Display *xdpy, Drawable draw, const unsigned char *data,
-		  int w, int h, int colors)
-{
-  int j, i;
-  unsigned char	ialpha;
-  Pixmap pix;
-  int bitmapSize = ((w + 7) >> 3) * h; // (w/8) rounded up times height
-  char *aData = calloc(1, bitmapSize);
-  char *cData = aData;
-
-  if (colors == 4)
-    {
-      int k;
-      for (j = 0; j < h; j++)
-	{
-	  k = 0;
-	  for (i = 0; i < w; i++, k++)
-	    {
-	      if (k > 7)
-		{
-	      	  cData++;
-	      	  k = 0;
-	      	}
-	      data += 3;
-	      ialpha = (unsigned short)((char)*data++);
-	      if (ialpha > ALPHA_THRESHOLD)
-		{
-		  *cData |= (0x01 << k);
-		}
-	    }
-	  cData++;
-	}
-    }
-  else
-    {
-      for (j = 0; j < bitmapSize; j++)
-	{
-	  *cData++ = 0xff;
-	}
-    }
-
-  pix = XCreatePixmapFromBitmapData(xdpy, draw, (char *)aData, w, h,
-				    1L, 0L, 1);
-  free(aData);
-  return pix;
-}
+#define ALPHA_THRESHOLD 158
```

Comment:
I really would prefer to move this define before the first usage of the value. This may work nowadays but I am still used to the old way :-(

Reply:
Actually this is a duplicate. The other is at line 3105. Removed.